portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,25 +20,33 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.sps.data.User;
 import com.google.gson.Gson;
 import java.util.List;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  
+  ArrayList<User> users = new ArrayList<User>();
```

Comment:
prefer using the List interface on the left side of the assignment.
Also! One of my favorite recent(ish) changes to the java language is that you can now skip the type specification on the right side. eg:
List\<User\> = new ArrayList<>();

full details on when this is allowed are here: https://docs.oracle.com/javase/8/docs/technotes/guides/language/type-inference-generic-instance-creation.html